libs/financial-aid/shared/src/components/FileList/FileList.tsx

```diff
@@ -18,18 +28,35 @@ const FileList = ({ className, files }: Props) => {
   if (files === undefined || files.length === 0) {
     return null
   }
+
+  const [openFile] = useLazyQuery(GetSignedUrlQuery, {
+    fetchPolicy: 'no-cache',
+    onCompleted: (data: { getSignedUrlForId: { url: string } }) => {
+      window.open(data.getSignedUrlForId.url, '_blank')
+    },
+    onError: (error) => {
+      // TODO: What should happen here?
+      console.log(error)
+    },
+  })
+
   return (
     <Box className={cn({ [`${className}`]: true })} marginBottom={2}>
       <>
         {files.map((item, index) => {
           return (
-            <a
+            <Box
```

Comment:
div with an onclick event, isnt that bad practice? why not have it a tag ? Just curious👀

Reply:
True 💯 will change